extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaQuarkusConfiguration.java

```diff
@@ -0,0 +1,147 @@
+
+    //    @ConfigItem(name = "CoreEnvironmentBean.nodeIdentifier", defaultValue = "1")
+    //    public String nodeIdentifier;
+    /**
+     * config
+     */
+    @ConfigItem(name = "JTAEnvironmentBean.xaRecoveryNodes", defaultValue = "1")
+    public List<String> xaRecoveryNodes;
+    /**
+     * config
+     */
+    @ConfigItem(name = "JTAEnvironmentBean.xaResourceOrphanFilterClassNames", defaultValue = "com.arjuna.ats.internal.jta.recovery.arjunacore.JTATransactionLogXAResourceOrphanFilter,"
+            + "com.arjuna.ats.internal.jta.recovery.arjunacore.JTANodeNameXAResourceOrphanFilter,"
+            + "com.arjuna.ats.internal.jta.recovery.arjunacore.SubordinationManagerXAResourceOrphanFilter")
+    public List<String> xaResourceOrphanFilterClassNames;
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoreEnvironmentBean.socketProcessIdPort", defaultValue = "0")
+    public int socketProcessIdPort;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.recoveryModuleClassNames", defaultValue = "com.arjuna.ats.internal.arjuna.recovery.AtomicActionRecoveryModule,"
+            + "com.arjuna.ats.internal.txoj.recovery.TORecoveryModule,"
+            + "com.arjuna.ats.internal.jta.recovery.arjunacore.SubordinateAtomicActionRecoveryModule,"
+            + "com.arjuna.ats.internal.jta.recovery.arjunacore.XARecoveryModule")
+    public List<String> recoveryModuleClassNames;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.expiryScannerClassNames", defaultValue = "com.arjuna.ats.internal.arjuna.recovery.ExpiredTransactionStatusManagerScanner")
+    public List<String> expiryScannerClassNames;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.recoveryPort", defaultValue = "4712")
+    public int recoveryPort;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.recoveryAddress")
+    public String recoveryAddress;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.transactionStatusManagerPort", defaultValue = "0")
+    public int transactionStatusManagerPort;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.transactionStatusManagerAddress")
+    public String transactionStatusManagerAddress;
+    /**
+     * config
+     */
+    @ConfigItem(name = "RecoveryEnvironmentBean.recoveryListener", defaultValue = "true")
+    public boolean recoveryListener;
+
+    // non final properties
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.readonlyOptimisation", defaultValue = "true")
+    public boolean readonlyOptimisation;
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.startDisabled", defaultValue = "false")
+    public boolean startDisabled;
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.defaultTimeout", defaultValue = "60")
```

Comment:
is it expected value when we are used to with 5 minutes in Narayana/WFLY?

Reply:
Same comment as before: do you think I should be taking most of the defaults from what we set in the wildfly transactions subsystem?

Reply:
+1